src/styles/_page.scss

```diff
@@ -90,3 +90,22 @@ section[role='tabpanel'] [class*='--col'] + [class*='--col'] {
 .tab--columns_label {
   margin-bottom: 0.5rem;
 }
+
+//---------------------------------------
+// Tooltip
+//---------------------------------------
+
+.tooltip-trigger {
+  @include button-reset.reset();
```

Comment:
looks like it's failing bc of this. do we need to include it at the top? @tay1orjones

Reply:
I think so, yes. `*.module.scss` files are provided a context already containing the utility modules like button, theme, convert, etc.

`.scss` files without `module` are not provided that context and instead need to `@use` the modules needed.

Reply:
maybe better i move this to a module then -- can do 👍🏾